### PR TITLE
[Merged by Bors] - feat: Port/Data.Polynomial.Cardinal

### PR DESCRIPTION
**Pull request by casavaca**
Rename only.

---

- [ ]  depends on: #2620 
[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#2620
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#2620~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/Data/Polynomial/Cardinal.lean [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Polynomial.Basic
+import Mathlib.SetTheory.Cardinal.Ordinal
+
+/-!
+# Cardinality of Polynomial Ring
+
+The reuslt in this file is that the cardinality of `R[X]` is at most the maximum
```
Comment (jcommelin) [inline Mathlib/Data/Polynomial/Cardinal.lean:17]:
```suggestion
The result in this file is that the cardinality of `R[X]` is at most the maximum
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**